Docs.Site/Docs/Release-Notes/MyGet-2017.2.markdown

```diff
@@ -0,0 +1,123 @@
+# MyGet 2017.2 Release Notes
+
+MyGet 2017.2 was released on **TODO**, 2017.
+
+## Highlights
+
+Next to some new [features](#Features) and many [fixes](#Bug_Fixes_amp_Other_Improvements), this 2017.2 release of MyGet again adds some new functionality to the service.
+
+Major highlights of this release are: 
+
+* We added **PHP Composer** support, and welcome PHP developers to the MyGet family! ([Announcement](https://blog.myget.org/post/2017/09/11/php-composer-packages-on-myget.html) | [Docs](docs/walkthrough/getting-started-with-php-composer))
```

Comment:
This also resulted in a bug fix on Composer itself (which is now merged, yay!) - https://github.com/composer/composer/pull/6717 will be part of Composer 1.5

Reply:
Right! Adding a note.